app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
```

Comment:
First and foremost, break this code down into functions and call it at the end of the file. Without the definition of a function it's just a bunch of a script that does not make any sense.

At this moment your code is not readable and it is impossible to debug it since you won't be able to handle errors properly.

Each function has its own purpose and logic, so you can reuse it in different places with different parameters. 
Make sure you consider it when you make changes